src/styles/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  width: 200px;
+  border: 1px solid $light-white-color;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 32px 16px 16px;
+  margin-right: 48px;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.